packages/block-editor/src/components/inserter/menu.js

```diff
@@ -276,7 +272,7 @@ function InserterMenu(
 						prioritizePatterns={ prioritizePatterns }
 						onSelect={ handleSetSelectedTab }
 					>
-						{ getCurrentTab }
+						{ inserterTabsContents }
```

Comment:
This is nitty, but in principle, I find it unusual to pass an object as the `children`. I considered two alternative approaches:

- keep using the `children` prop, but passing `Tabs.Tab` components (although this doesn't seem convenient, since all the tabs data is in the `InserterTabs` component)
- instead of using `children`, use a new different prop (eg. `tabNames` or something similar).

The second option seems like the most viable. We should be able to make API changes to `InserterTabs` since it doesn't seem to be used outside of this folder.

Reply:
I can get behind that! I also looked and don't see this component being used elsewhere, so I've gone with the second option, passing the tab contents object via a new `tabsContents` prop instead of passing it as `children`.

Let me know what you think 🙂 5fe8b6797d

Reply:
Looks good!